### PR TITLE
Install.rst: Add note for virtualenv creation

### PR DESCRIPTION
**Pull request by diveshuttam**
This note helps the users who have both python2 and python3 installed.
Adds information to setup python3 only virtualenv.

Closes https://github.com/coala/documentation/issues/515

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by damian1996** [comment]
You should change Fixes to Closes in commit message :)

File: Users/Install.rst [hunk 1]
```diff
@@ -106,6 +106,13 @@ folder:
 
     $ virtualenv venv
 
+.. note::
+
+    If you have both Python 3 and Python 2 installed, use
```
Comment (newbazz) [inline Users/Install.rst]: make`python 3` to `python3` and similarly to python 2

File: Users/Install.rst [hunk 2]
```diff
@@ -106,6 +106,13 @@ folder:
...
+    ``virtualenv venv -p $(which python3)``.
+    This creates an isolated Python 3 environment called venv
```
[the remark this answers is not in the document]
Comment (anctartica) [reply Users/Install.rst]: we probably don't want this as `venv` is created here only for using coala whereas in the development setup notes `coala-venv` is created for the purpose of development.

[the remark this answers is not in the document]
Comment (realslimshanky) [reply Users/Install.rst]: @anctartica actually, naming the virtual environment doesn't change its functionality. Using either `venv` or `coala-venv` is correct.

[the remark this answers is not in the document]
Comment (anctartica) [reply Users/Install.rst]: yes, i know.. was referring to the context in which the name is used in the documentation

[the remark this answers is not in the document]
Comment (diveshuttam) [reply Users/Install.rst]: @pareksha @realslimshanky the name `venv` is being used in context of this document. Therefore, I felt using `venv` is better. Using `coala-venv` would require changes at other places in the document also. 
If you want, I can add another commit changing `venv` to `coala-venv`.

[the remark this answers is not in the document]
Comment (pareksha) [reply Users/Install.rst]: Since it's ``venv`` in the rest documentation, use ``venv`` only. Didn't note that before. ;)

[the remark this answers is not in the document]
Comment (newbazz) [reply Users/Install.rst]: @diveshuttam yea i just had a look and IMO you can carry on with`venv` and later on file an issue to change it after asking a maintainer ;)

**Comment by diveshuttam** [comment]
@Kagamihime Oh forgot to change the comment here. Had already changed the commit by looking at another PR.

File: Users/Install.rst (hunk 2 above)
Comment (newbazz) [inline Users/Install.rst]: btw change this `called venv`.

File: Users/Install.rst [hunk 3]
```diff
@@ -106,6 +106,13 @@ folder:
...
+    in your current directory, as coala only works for Python >= 3.4
```
Comment (newbazz) [inline Users/Install.rst]: Add a period here.

File: Users/Install.rst (hunk 1 above)
Comment (diveshuttam) [reply Users/Install.rst]: @newbazz are you referring to case also??

Comment (newbazz) [reply Users/Install.rst]: nope ;P my bad

File: Users/Install.rst (hunk 2 above)
Comment (diveshuttam) [reply Users/Install.rst]: @newbazz I have changed it to:
 Python3 environment named ``venv`` 
 Does it seem fine??

[the remark this answers is not in the document]
Comment (realslimshanky) [reply Users/Install.rst]: @pareksha @newbazz @diveshuttam I agree with you guys on being consistent with the naming convention. However my comment was only regarding functionality either way :blush:

File: Users/Install.rst [hunk 4]
```diff
@@ -106,6 +106,13 @@ folder:
 
     $ virtualenv venv
 
+.. note::
+
+    If you have both Python3 and Python2 installed, use
```
Comment (Makman2) [inline Users/Install.rst]: You should add a space between "Python" and its version, so it doesn't count as an own name.

File: Users/Install.rst [hunk 5]
```diff
@@ -106,6 +106,13 @@ folder:
...
+    ``virtualenv venv -p $(which python3)``.
+    This creates an isolated Python3 environment named ``venv``
```
Comment (Makman2) [inline Users/Install.rst]: here too

File: Users/Install.rst [hunk 6]
```diff
@@ -106,6 +106,13 @@ folder:
...
+    in your current directory, as coala only works for Python >= 3.4 .
```
Comment (Makman2) [inline Users/Install.rst]: 1. I would rephrase and try to avoid mathematical notations:
   "... works for Python 3.4 and above."
2. No space between `3.4` and the full stop (automatically solved when you rephrase like suggested)

**Comment by Makman2** [comment]
unack 5dd32a995f41542b20139a881adff68d2afb47a4

File: Users/Install.rst (hunk 4 above)
Comment (newbazz) [reply Users/Install.rst]: Dont we always write it as `Python3 and Python2`?
P.S it was separated earlier but i asked to remove the space ;P

**Comment by Makman2** [comment]
ack 729c1a06281c4f578426baf91800e6ba3c71e7ae

**Comment by Makman2** [comment]
@gitmate-bot ff
